column.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/alexbrainman/odbc/api"
+	"github.com/edharper01/odbc/api"
```

Comment:
You shouldn't rename import path. Otherwise users of this package will end-up using source from your github repo.

See, for example, these
https://splice.com/blog/contributing-open-source-git-repositories-go/
http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html
on how to create pull request without changing import path.